database/test_database.go

```diff
@@ -31,6 +33,17 @@ var (
 		TestMemorySafetyDatabase,
 		TestMemorySafetyBatch,
 	}
+
+	// Benchmarks is a list of all database benchmarks
```

Comment:
Can we move the benchmarks to `benchmark_database.go` to separate performance tests and correctness tests?

Reply:
Benchmarks were moved to `benchmark_database.go`.